app/views/course/assessment/assessments/_edit.json.jbuilder

```diff
@@ -1,10 +1,10 @@
 # frozen_string_literal: true
 json.attributes do
   json.(@assessment, :id, :title, :description, :start_at, :end_at, :bonus_end_at, :base_exp,
-    :time_bonus_exp, :published, :autograded, :show_private, :show_evaluation, :skippable,
-    :tabbed_view, :view_password, :session_password, :delayed_grade_publication, :tab_id,
-    :use_public, :use_private, :use_evaluation, :allow_partial_submission, :has_personal_times,
-    :affects_personal_times, :show_mcq_answer)
+        :time_bonus_exp, :published, :autograded, :show_private, :show_evaluation, :skippable,
```

Comment:
Sorry one last comment - what happened to the spacing for this part?

Reply:
houndci-bot was complaining about the alignment, see here:
https://github.com/Coursemology/coursemology2/pull/4027#discussion_r692689310